hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -95,6 +95,43 @@
       width:   "1"
     }
   ],
+  features: [
+    { name: "AON_TIMER.WAKEUP",
```

Comment:
I would break this into a few sub-features, like AON_TIMER.WAKEUP.INTERRUPT, AON_TIMER.WAKEUP.WAKEUP_REQUEST, and AON_TIMER.WAKEUP.CONFIG.